config/galaxy.ini.sample

```diff
@@ -213,6 +213,19 @@ paste.app_factory = galaxy.web.buildapp:app_factory
 # if it cannot find a local copy and conda_exec is not configured.
 #conda_auto_init = False
 
+# Certain dependency resolvers (namely Conda) take a considerable amount of
+# time to build an isolated job environment in the job_working_directory if the
+# job working directory is on a network share.  Set the following option to True
+# to cache the dependencies in a folder. This option is beta and should only be
+# used if you experience long waiting times before a job is actually submitted
+# to your cluster. If you activate this option and install or remove dependencies,
+# you may need to clear out old cached environments
+#use_cached_dependency_manager = False
```

Comment:
In principle this works for all non-container resolver types, except that you get the biggest benefits with conda. But maybe I should find another angle and do the caching only for conda ... I kind of like inheriting from DependencyManager though.
Let's see if I can come up with an elegant way to do this for conda only. That would at least reduce the number of environments that would be cached.